subprojects/docs/src/samples/userguide/java/basic/kotlin/build.gradle.kts

```diff
@@ -36,12 +36,12 @@ tasks.getByName<Test>("test") {
 // tag::practical-integ-test-source-set[]
 sourceSets {
     create("intTest") {
-        compileClasspath += sourceSets["main"].output
-        runtimeClasspath += sourceSets["main"].output
+        compileClasspath += sourceSets.main.get().output
```

Comment:
:o: I would take advantage of kotlin here and dedupe the complex output expression:

```kotlin
sourceSets.main.get().output.let {
   runtimeClasspath += it
   compileClasspath += it
}
```